### DISCUSSION TITLE
Support for Redis version 7.2.x

### DISCUSSION THREAD

**Discussion by alita1991** [OT-CONTAINER-KIT/redis-operator#890] [General]
Hi, I looked over the documentation and found out that the latest supported Redis version is 7.0.12.

When is expected for the operator to support version 7.2.x?

Thanks

**Comment by danielloader** [comment]
or 7.4.x I guess?

**Comment by discostur** [comment]
Or 8.x ... i saw that there are images in the docker registry already
